comp/process/runner/component.go

```diff
@@ -32,8 +32,3 @@ type Mock interface {
 var Module = fxutil.Component(
 	fx.Provide(newRunner),
 )
-
-// MockModule defines the fx options for the mock component.
-var MockModule = fxutil.Component(
```

Comment:
I ended up removing this in the end. I couldn't think of any tests where it would be useful to use a mock instead of the real thing, since the runner isn't a dependency of anything, and has no special method outside of `Run`, `Stop` and `GetChecks`, which isn't currently used. I will add it back in if it becomes useful in the future.